src/services.py

```diff
@@ -677,9 +666,10 @@ def get_other_party_details(self, chat_room_id, user_id):
 
 
 class SocialLogin:
-    def __init__(self, config, sio):
+    def __init__(self, config, sio, UserService=UserService):
         self.config = config
         self.sio = sio
+        self.UserService = UserService
```

Comment:
Let's get rid of this `self.UserService` etc injection stuff. I felt that it's not very useful